src/main/java/com/osp/dragon/service/ArticleService.java

```diff
@@ -14,18 +16,19 @@
 @Service
 public class ArticleService {
 
-    private final Map<String, Article> articlesMap;
 
-    public ArticleService() {
-        this.articlesMap = initializeArticlesCache();
+    List<Article> articlesList;
+
+    public ArticleService() throws IOException, ClassNotFoundException {
+        this.articlesList = initializeArticles();
```

Comment:
jeżeli tak zrobiłeś to ten obiekt zawsze będzie miał te same artykuły, nigdy nie będą zaupdatowane

Reply:
Powinieneś pobierać artykuły tam gdzie je wywołujesz

Reply:
Żeby zawsze czytało świeże wartości

Reply:
nie dodałem tego, bo nie byłem pewny,bo jak danej funkcji wywojuje pobranie artykułów, to każda ta funkcja rzucałaby wyjątki, tak może być ?